README.md

```diff
@@ -11,11 +11,11 @@ NPM-based tool to validate App Builder templates during submission process to [A
 
 ## install.yml file
 
-| Field        | Description                                    | Required           |
-|--------------|------------------------------------------------|--------------------|
-| `categories` | Template must have categories defined          | :heavy_check_mark: |
-| `extension`  | Template must declare extension points it uses | :heavy_check_mark: |
-| `services`   | Template must declare services it uses         | :heavy_check_mark: |
+| Field        | Description                                                                                                | Required           |
+|--------------|------------------------------------------------------------------------------------------------------------|--------------------|
+| `categories` | Template must have categories defined                                                                      | :heavy_check_mark: |
+| `extension`  | Contains the extension metadata, like `serviceCode`.                                                       |                    |
```

Comment:
Suggestion:
Template might declare extension points it uses. Should contain the extension metadata, like `serviceCode`

Reply:
applied, thank you